stacks/environment-azure/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "password" {
+    value = random_password.password.result
+}
```

Comment:
Probably a good idea to mark the output as sensitive. It will obscure the value in the terraform output.